File: Challenge 2.py
def getlongestword(sentence):
    text = sentence.split()
    return max(len(word) for word in text)

print(getlongestword("The quick brown fox"))  
print(getlongestword("Hello world"))
print(getlongestword("I love coding"))
print(getlongestword("A"))
print(getlongestword("Supercalifragilisticexpialidocious is long"))

File: hub_round2/challenge2.js
let sentence = "The quick brown fox jumps over the lazy dog";
function findlongestWord(str) {
  let words = str.split(" ");
  let longestWord = "";
  for (let word of words) {
    if (word.length > longestWord.length) {
      longestWord = word;
    }
  }
  return longestWord;
}

let longestWordInSentence = findlongestWord(sentence);
console.log(longestWordInSentence);
